test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -1679,6 +1680,155 @@ public void testGetDocContentSubSiteUniquePermissions() throws Exception {
   }
 
   @Test
+  public void testGetDocContentSubSiteUniquePermissionsInvalidUser()
```

Comment:
It handles 2 scenarios :1 Exception during siteuser mapping 2. Exception during refresh siteuser mapping.  atomicNumberGetSiteUserMappingCalls gives the count for siteusermapping calls.